src/subscriptions/index.js

```diff
@@ -256,20 +256,72 @@ export default async function startSubscriptions() {
     // get initial state populated
     AccountSub.update()
     BlockSub.update()
+  NetworkSub.update()
 
     // create filter listening for latest new blocks
-    const subscription = Web3.web3WS.eth.subscribe('newBlockHeaders')
-
-    subscription.on('data', (blockHeader) => {
-        console.debug('New block header - updating AccountSub, BlockSub + subscribers', blockHeader.timestamp)
-        AccountSub.update()
+  const subscription = Web3.web3WS.eth.subscribe("newBlockHeaders")
+
+  subscription.on("data", (blockHeader) => {
+    console.debug(
+      "New block header - updating AccountSub, BlockSub + subscribers",
+      blockHeader.timestamp,
+    )
+    // AccountSub.update()
         BlockSub.update()
     })
 
-    subscription.on('error', (err) => {
-        console.error('An error in newBlockHeaders WS subscription occurred - unsubscribing.', err.message || err)
+  subscription.on("error", (err) => {
+    console.error(
+      "An error in newBlockHeaders WS subscription occurred - unsubscribing.",
+      err.message || err,
+    )
         subscription.unsubscribe()
     })
 
-    return () => subscription && subscription.unsubscribe()
+  const interval = setInterval(() => AccountSub.update(), 2000)
```

Comment:
I don't really like this as it defeats the purpose of the websockets and not pinging infura over and over. Can we put this conditionally for development/localhost?